.circleci/config.yml

```diff
@@ -72,11 +72,14 @@ jobs:
     steps:
```

Comment:
Nit: maybe rename the job to `tests_py39_coverage_docs` for more clarity from Github PoV